src/components/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -14,9 +14,16 @@
  */
 
 import React, { HTMLProps, Ref } from 'react';
+import { css } from '@emotion/core';
 
+import styled, { StyleProps } from '../../styles/styled';
 import { uniqueId } from '../../util/id';
 import { RadioButton, RadioButtonProps } from '../RadioButton/RadioButton';
+import Label from '../Label';
+import {
+  InlineMessage,
+  InlineMessageProps,
+} from '../InlineMessage/InlineMessage';
```

Comment:
The InlineMessage component is meant for _form_ level errors. _Field_ level errors should use the [ValidationHint](https://github.com/sumup-oss/circuit-ui/tree/canary/src/components/ValidationHint) component.

Reply:
Yeah I placed the ValidationHint component but then Bobi (our designer) said that it's better to place the InlineMessage. 🤷

Reply:
What was his reason for it? 

This simply goes against the usage guidelines for the InlineMessage. It's for form level errors inside a card only. I know that it's being used differently/incorrectly in the Dashboard and Shop currently, we're trying to correct those cases as well. 

Please use the ValidationHint.